5-flexbox/assets/styles.css

```
@@ -0,0 +1,72 @@
+.content {
+    display: flex;
+    flex-direction: row;
+}
+
+img {
+    width: 300px;
+    height: 300px;
```

```suggestion
    width: 300px;
    height: 300px;
```
Estas propiedades es mejor setearlas en los atributos de la imagen para que no sea necesario que el navegador descargue el CSS para poder mostrar el contenido de la misma.